CODEOWNERS

```diff
@@ -135,14 +135,13 @@ Kconfig*                                  @tejlmand
 /modules/                                 @tejlmand
 /modules/hostap/                          @krish2718 @jukkar @rado17 @sachinthegreen @rlubos
 /modules/mcuboot/                         @de-nordic @nordicjm
-/modules/cjson/                           @simensrostad @plskeggs @sigvartmh
+/modules/cjson/                           @nrfconnect/ncs-cia @plskeggs @sigvartmh
 /modules/tfm/                             @frkv @Vge0rge @vili-nordic @joerchan @SebastianBoe @mswarowsky
 /modules/trusted-firmware-m/              @frkv @Vge0rge @vili-nordic @joerchan @SebastianBoe @mswarowsky
 /samples/                                 @nrfconnect/ncs-test-leads
-/samples/net/mqtt/                        @simensrostad
-/samples/net/udp/                         @simensrostad
-/samples/sensor/bh1749/                   @gregersrygg
-/samples/sensor/bme68x_iaq/               @maxd-nordic
+/samples/net/                             @nrfconnect/ncs-cia
```

Comment:
@lemrey Is this OK by you? It was suggested by @simensrostad , and we've taken over the ownership of the tests. I think it makes sense that we then also own the code itself.

Reply:
Please add me as well